### ISSUE TITLE
Feature Proposal: no [OverrideBuiltins]

### ISSUE THREAD

**Issue by domenic** [WICG/feature-policy#251]
See https://github.com/whatwg/html/issues/4112. The case people are most concerned about are when you could introduce an element with a name or ID that overrides the built-ins, i.e. the [OverrideBuiltins] cases on Document and HTMLFormElement.

You could imagine various expansions of this, e.g.

- Turning off named getters/setters entirely. (Although I'm not sure we have replacements in 100% of cases)
- Turning off "bad legacy DOM and HTML APIs" entirely. (E.g. things like Attr nodes.)

**Comment by ojanvafai** [comment]
I wonder if we could generalize it to also remove the global scope polluter? (just thinking out loud, that might by nonsensical)

**Comment by domenic** [comment]
Yeah, that's kind of covered under my "turning off named getters/setters entirely" case, although it's a particularly egregious one.

**Comment by annevk** [comment]
Entirely would be bad for `dataset`, but for cases with "fallbacks" (and therefore clashes) this makes a lot of sense.

I'm not convinced optional `Attr` nodes will help much.
